css/style.css

```diff
@@ -0,0 +1,228 @@
+body{
+   margin: 0;
+   padding: 0;
+}
+.container{
+  margin-left: 42px;
+    margin-right: 100px;
+}
```

Comment:
These margins create spaces at both ends, is this needed?

Reply:
Oh okay thanks for the review let do it